apps/web-tools/trustchain/components/Tick.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react";
+
+export function Tick({ timestamp }: { timestamp: number }) {
```

Comment:
We should handle translations for this component

Reply:
not for web tools.